libs/community/langchain_community/chat_message_histories/sql.py

```diff
@@ -80,36 +92,85 @@ def get_sql_model_class(self) -> Any:
         return self.model_class
 
 
+DBConnection = Union[AsyncEngine, Engine, str]
+
+
 class SQLChatMessageHistory(BaseChatMessageHistory):
     """Chat message history stored in an SQL database."""
 
+    @property
+    def Session(self) -> Union[scoped_session, async_sessionmaker]:
+        logger.warning("Session is deprecated, use session_maker instead")
+        return self.session_maker
+
     def __init__(
         self,
         session_id: str,
-        connection_string: str,
+        connection_string: Optional[str] = None,
+        connection: Union[None, DBConnection] = None,
```

Comment:
This would be a breaking change for any users that used positional arguments to define the table_name.

Instead, could you place all the new named arguments at the end of the function signature and force them to be invoked as named arguments by users using the `*` operator.

Hopefully ruff ends a lint rule for this at some point. :)

Reply:
done